doc/faq.rst

```diff
@@ -244,3 +243,12 @@ program: Insert the following instructions in your main script::
 
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
+
+How do I set a random_state for an entire execution?
+----------------------------------------------------
+
+For testing and replicability, it is often important to have the entire execution
+controlled by a single seed for the pseudo-random number generator used in algorithms
+that have a randomized component. While there is no ``random.seed(seed)`` function,
+scikit-learn uses the global random states by default and one could thus define a
+global random state that would be used throughout an execution.
```

Comment:
I think that you should be more explicit here and give a specific example:
<pre>
>>> import numpy as np
>>> np.random.seed(42)
</pre>